tests/scripts/thread-cert/border_router/test_firewall.py

```diff
@@ -301,6 +307,16 @@ def verify(self, pv: pktverify.packet_verifier.PacketVerifier):
         _pkt = pkts.filter_wpan_src64(vars['BR_1']).filter_ping_request().must_next()
         pkts.filter_wpan_src64(vars['Router_1']).filter_ping_reply(identifier=_pkt.icmpv6.echo.identifier).must_next()
 
+        # 16. Host sends a UDP packet to router1's OMR address's TMF port (61631).
+        pkts.filter_eth_src(vars['Host_ETH']).filter_ipv6_dst(vars['Router_1_OMR'][0]).filter(
+            lambda p: p.udp.dstport == 61631 and p.udp.length == len("HELLO") + 8).must_next()
+        pkts.filter_wpan_src64(vars['BR_1']).filter_ipv6_dst(vars['Router_1_OMR'][0]).filter(
+            lambda p: p.udp.dstport == 61631 and p.udp.length == len("HELLO") + 8).must_not_next()
+
+        # 17. BR1 sends a UDP packet to router1's ML-EID's TMF port (61631).
+        pkts.filter_wpan_src64(vars['BR_1']).filter_ipv6_dst(vars['Router_1_MLEID']).filter(
+            lambda p: p.udp.dstport == 61631 and p.udp.length == len("BYE") + 8).must_not_next()
```

Comment:
I realized that it's a bit hard for me to understand what is testing here. Could you update the comment to include the expected behavior?